demo/src/main/java/com/novoda/landingstrip/SimpleTextTabActivity.java

```diff
@@ -39,8 +39,8 @@ public void onTabClicked(int position, Data data) {
             }
         };
         ExampleTabAdapter tabAdapter = new ExampleTabAdapter(getLayoutInflater(), setCurrentItemOnClick, data);
-
-        viewPager.addOnPageChangeListener(landingStrip);
+        ViewPager.OnPageChangeListener landingStripPageChanger = LandingStripPageChangeListener.create(landingStrip);
+        viewPager.addOnPageChangeListener(landingStripPageChanger);
```

Comment:
the `landingstrip` class is pretty empty 121 lines~, almost all logic has been delegated out of it